cibyl/outputs/cli/ci/system/impls/zuul/colored.py

```diff
@@ -213,8 +215,95 @@ def print_build(self, build: Build) -> str:
                 if build.duration.value:
                     result.add(get_duration_section(self.palette, build), 1)
 
+            if self.query >= QueryType.TESTS:
+                result.add(self.palette.blue('Test Suites: '), 1)
```

Comment:
Shouldn't we add a guard here to catch the cases of jobs that have no builds? Like we do with pipelines and projects. Also, in those cases I would add a message like `No tests found in query`

Reply:
You are right, I am missing that.

Reply:
Done.